packages/astro-prism/package.json

```diff
@@ -13,8 +13,13 @@
   "homepage": "https://astro.build",
   "main": "index.mjs",
   "scripts": {},
+  "files": [
+    "components"
+  ],
   "exports": {
-    ".": "./index.mjs"
+    ".": "./index.mjs",
+    "./components": "./components/index.js",
```

Comment:
Nit, but just `@astrojs/prism/component` with no (s)? There's not going to be multiple of these are there?

Reply:
Update: changed to a default export:

```js
import Prism from '@astrojs/prism/component';
```

Reply:
Yeah i was thinking that from the main would be nice but can't really do that, too bad. This works fine though, thanks!